src/app/Commands/CreateUser.php

```diff
@@ -45,7 +45,10 @@ public function handle()
         $count = (int) $this->ask('How many would you like to create?');
 
         while ($count > 0) {
+
+
             if (!$this->option('fake')) {
+
```

Comment:
I know you'll think this is picky, but lets get rid of the extra whitespace here :).